.gitignore

```diff
@@ -4,3 +4,6 @@
 integration/tls/cert
 resources
 timestream-prometheus-connector
+darwin
+linux
+windows
```

Comment:
Please make sure all files end with a newline. This will likely involve your text editor configuration.